backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHIteratorApi.scala

```diff
@@ -179,7 +180,6 @@ class CHIteratorApi extends IIteratorApi with Logging with LogLevelUtil {
 
       override def next(): Any = {
         val cb = resIter.next()
-        updateOutputMetrics(1, cb.numRows())
```

Comment:
Hi @zzcclp, is this function call still needed on CH backend?

Reply:
remove this first, if there is any issue, I will fix it later.